src/fides/cli/utils.py

```diff
@@ -117,15 +147,24 @@ def check_and_update_analytics_config(ctx: click.Context, config_path: str) -> N
             user={"analytics_opt_out": ctx.obj["CONFIG"].user.analytics_opt_out}
         )
 
-    is_analytics_opt_out = ctx.obj["CONFIG"].user.analytics_opt_out
+    if ctx.obj["CONFIG"].user.analytics_opt_out is False and not is_user_registered(
```

Comment:
What happens if there is no server running for this? I'd expect the GET to error, so this whole thing likely crashes right?

Reply:
This helper is called behind the server check: https://github.com/ethyca/fides/blob/b40ec0a838eb3db8fb7ec330125cf5844bebcdd3/src/fides/cli/__init__.py#L90

Reply:
OK, I see that guard now... looking through it though, there are several other commands that aren't considered "API commands" beyond just init and deploy: https://github.com/ethyca/fides/blob/b40ec0a838eb3db8fb7ec330125cf5844bebcdd3/src/fides/cli/__init__.py#L23

That'd include even just a basic `fides view config` done locally without a server, for example.